src/components/modal/modal-buttons/modal-buttons.tsx

```diff
@@ -0,0 +1,16 @@
+import { Component, ComponentInterface, h } from '@stencil/core';
+
+@Component({
+  tag: 'bds-modal-buttons',
```

Comment:
Please, change the tag 'bds-modal-buttons' for 'bds-modal-action' and all places its show.